src/dagmc/DagMC.hpp

```diff
@@ -225,10 +225,6 @@ class DagMC {
 
   ErrorCode point_in_volume_slow(const EntityHandle volume, const double xyz[3],
                                  int& result);
-#if MOAB_VERSION_MAJOR == 5 && MOAB_VERSION_MINOR > 2
-  ErrorCode find_volume(const double xyz[3], EntityHandle& volume,
-                        const double* uvw = NULL);
-#endif
```

Comment:
This is a regression and removes code that was intentionally added by a subsequent PR.